packages/@o3r/core/README.md

```diff
@@ -116,3 +116,15 @@ It includes the following modules:
 * @o3r/configuration
 * @o3r/dynamic-content
 * @o3r/rules-engine
+
+### External (Custom) Preset
+
+The preset mechanism is supporting, via the `--external-preset` option, presets defined by module providers themself.
+
+The modules can be regrouped in custom Presets by following the next steps:
+
+* Adding a **keyword**, in the module *package.json*'s `keywords` field, following the pattern `otter-preset-<my-preset>`.
+* The module needs to be published to public (or private) NPM Registry.
+* The package's scope should be whitelisted, via a Pull Request, in the following [OTTER_MODULE_SUPPORTED_SCOPES constant](https://github.com/AmadeusITGroup/otter/blob/main/packages/@o3r/schematics/src/modules/modules.constants.ts).
```

Comment:
I know we already discussed about this one, but that means that as long as we have whitelisted them, they can push for any repository on their side without any approval from us ? Is this a good idea? 
I would at least put a disclaimer in the case there are packages that have not been published under our scope.